wg-naming/README.md

```diff
@@ -0,0 +1,80 @@
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+--->
+# Naming Working Group
+
+
+## Stakeholder SIGs
+* SIG Architecture
+* SIG Contributor Experience
+* SIG Docs
+
+
+
+## Organizers
+
+* Celeste Horgan (**[@celestehorgan](https://github.com/celestehorgan)**), CNCF
+* Jaice Singer DuMars (**[@jdumars](https://github.com/jdumars)**), Apple
+* Stephen Augustus (**[@justaugustus](https://github.com/justaugustus)**), VMware
+* Zach Corleissen (**[@zacharysarah](https://github.com/zacharysarah)**), Linux Foundation
+
+## Contact
+- Slack: [#wg-naming](https://kubernetes.slack.com/messages/wg-naming)
+- [Mailing list](https://groups.google.com/forum/#!forum/kubernetes-wg-naming)
+- [Open Community Issues/PRs](https://github.com/kubernetes/community/labels/wg%2Fnaming)
+<!-- BEGIN CUSTOM CONTENT -->
+
+**The following section will be reworked and formalized as a charter once the
+Working Group has been approved by the Steering Committee.**
+
+## Goals
+
+- Evaluate language and naming choices within the Kubernetes project, with
+  a specific initial focus of:
+  - Removing barriers to contribution and adoption by replacing harmful language with neutral terms whenever possible, including but not limited to language linked to racism, sexism, homophobia, transphobia, ableism, or discrimination against any protected or historically underrepresented group.
+  - Improving clarity of codebases and documentation by replacing idioms,
+    metaphors and slang specific to the English language
+- Create a list of harmful terms with proposed replacements
+- Define how any member of the Kubernetes project can
+  recommend language, how others can evaluate that proposal, and how to 
+  implement replacements across all codebases.
+  - Provide an easily findable location for language recommendations and
+    follow-up issues, similar to an architectural decision record
+  - Define long-term ownership of this process
+- Work with stakeholder SIGs to implement the changes recommended. We
+  anticipate the following:
+  - Provide stakeholder SIGs with guidance on naming, language conventions, and
+    processes
+  - Collaborate with SIG Architecture and other stakeholders on an
+    implementation timeline and strategies for dealing with follow-up issues
+    from renaming, like deprecations
+  - Collaborate with SIG Docs and SIG Contributor Experience on documenting
+    language recommendations and processes
+  - Work with the Code of Conduct Committee to add code architecture to the COC
+
+## Dissolution Criteria
+
+Once the Kubernetes community has:
+
+- A process in place to evaluate language changes on an ongoing basis
+- A binding list of terms to avoid in codebases across the project
+- A timeline on which to replace component names in the kubernetes/kubernetes
+  codebase
+- Defined long-term ownership of the policies and processes this WG creates
+
```

Comment:
non-blocking nit: whitespace

Reply:
@cblecker -- intentional. see the next line.